app/src/main/java/com/kickstarter/viewmodels/AccountViewModel.kt

```diff
@@ -72,18 +73,18 @@ interface AccountViewModel {
                 .compose(Transformers.neverErrorV2())
 
             userPrivacy
-                .map { it.me()?.chosenCurrency() }
+                .map { chosenCurrency() }
```

Comment:
Now this RXJava chain can be exchanged over to: 
` userPrivacy
                .map { it.chosenCurrency }
                .subscribe { this.chosenCurrency.onNext(it) }
                .addToDisposable(disposables)`

Reply:
Ah ok, yeah I wasn't sure what the ObjectUtils.coalesce was doing. Are you sure you want me to remove ```           .map { ObjectUtils.coalesce(it, CurrencyCode.USD.rawValue()) }```

Reply:
yeap, it's redundant now, if you look at `ObjectUtils,coalesce` internal code it basically adds the second parameter as default value if the first parameter is null